jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -9,6 +9,12 @@
 Supported by Red Hat.</description>
          <icon image48="images/redhat_48.png"></icon>
       </certification>
+      <certification id="com.jboss.jbds.discovery.certification.supported.notavailable" name="Supported - Not Available" url="https://devstudio.redhat.com/9.0/stable/updates/#central">
+         <description>Supported by Red Hat.
+
+This content is unavailable. Enable Early Access to see it.</description>
+         <icon image48="images/redhat_48.png"></icon>
```

Comment:
how does "supported.notavailable" imply "it is available on earlyaccess" ?

Where it is available should be in the description of the connector not in the certification. so something like "This feature set is currently unavailable for this version of Developer Studio. Please see the description for details on where/when it is available."